src/cmd/sql_cmd_test.cc

```diff
@@ -1969,6 +1969,27 @@ TEST_P(DBSDKTest, DeployStatsOnlyCollectDeployProcedure) {
     }
 }
 
+class StripSpaceTest : public ::testing::TestWithParam<std::pair<std::string_view, std::string_view>> {};
+
+std::vector<std::pair<std::string_view, std::string_view>> strip_cases = {
```

Comment:
how about "show\ncomponents\n;"

Reply:
statement str inside is handle by zetasql, should be fine.

anyway case like `show components; set val = ';    '` probably not work as expect